File: test/python/unittest-to-pytest/unconvertible/UNCONV-001.expected.py
# HAMLET-TODO [UNCONVERTIBLE-MODULE-SETUP]: Module-level setup/teardown has no direct pytest equivalent in-file
# Original: def setUpModule():
# Manual action required: Move to conftest.py as a session/module-scoped fixture
def setUpModule():
    print("module setup")

def test_example():
    assert True

File: test/java/testng-to-junit5/unconvertible/UNCONV-001.expected.java
import org.junit.jupiter.api.Test;

public class DependsOnTest {
    @Test
    public void testLogin() {
        assert true;
    }

    // HAMLET-TODO [UNCONVERTIBLE-DEPENDS-ON-METHODS]: TestNG dependsOnMethods has no JUnit 5 equivalent
    // Original: @Test(dependsOnMethods = {"testLogin"})
    // Manual action required: Refactor tests to be independent or use @Order annotation
    @Test(dependsOnMethods = {"testLogin"})
    public void testDashboard() {
        assert true;
    }
}
